src/ExpectedMessage.ts

```diff
@@ -160,7 +143,7 @@ export class ExpectedMessage {
             delete expectedResponse.agent;
 
             try {
-                expectWithDeepMatch(clone).to.deep.match(expectedResponse);
+                expect(clone).toDeepMatch(expectedResponse);
```

Comment:
To help make this cleaner, lets change this to be ```toDeeplyInclude``` and replace the [expectedResponseCollectionsAsIMessage loop](https://github.com/microsoftly/BotTester/pull/11/files#diff-38ae694041c82f7bcca679231dde7f1aL146) wtih 
``` javascript
expect(expectedResponseCollectionAsIMessage).toDeeplyInclude(expectedResponse)
```

Reply:
I made some of the changes. How would you handle the processing done to every message in the collection if we remove the forEach block ?